pandas/core/generic.py

```diff
@@ -11065,12 +11115,20 @@ def median(
             see_also=_stat_func_see_also,
             examples=_max_examples,
         )
-        def max(self, axis=None, skipna=True, level=None, numeric_only=None, **kwargs):
+        def max(
+            self,
+            axis: int | None | lib.NoDefault = lib.no_default,
```

Comment:
@simonjayhawkins is this going to render in the docs in a way we don't want?

(similar question bugs me in a WIP branch that changes the default from `method="pad"` in `.replace`)

Reply:
can be a followup